tests/devsim_layer_tests.py

```diff
@@ -363,10 +363,22 @@ def main():
         "vulkan_tools_install_dir",
         help="Path to the install directory of the KhronosGroup/Vulkan-Tools repo."
     )
+    parser.add_argument(
+        "--dir",
+        help="Specify the directory to run the tests in.  Defaults to current working directory.",
+        dest="directory"
+    )
 
     args = parser.parse_args()
 
     vulkaninfo_path = args.vulkan_tools_install_dir
+    working_directory = args.directory
+
+    starting_working_directory = os.getcwd()
+
+    if working_directory is not None:
+        working_directory = os.path.abspath(working_directory)
```

Comment:
It's easier to provide the value as a default argument:
```
parser.add_argument(
    "--dir",
    help="Specify the directory to run the tests in.  Defaults to current working directory.",
    dest="directory",
    default=os.getcwd()
)
```
Then after `parse_args()`:
```
working_directory = os.path.abspath(args.directory)
```

Reply:
Good suggestion!
Made the change and pushed it up.  Will merge after cloud CI passes.